examples/async_client_hello_callback/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "async_client_hello_callback"
+version = "0.1.0"
+authors = ["AWS s2n"]
+edition = "2021"
+
+[dependencies]
+s2n-quic = { version = "1", path = "../../quic/s2n-quic", features = ["provider-tls-s2n", "unstable_client_hello"]}
+tokio = { version = "1", features = ["full"] }
+s2n-tls = { version = "0", features = ["quic"] }
```

Comment:
I thought we exported s2n-tls from the integration crate

Reply:
not sure what the integration crate is?

Reply:
It's the s2n-quic-tls crate. But it looks like it wasn't exported: https://dnglbrstg7yg.cloudfront.net/2a4552a8aa9f0154f868fc026e6015a3c1ffb722/doc/s2n_quic/provider/tls/s2n_tls/index.html

The rustls crate was, though: https://dnglbrstg7yg.cloudfront.net/2a4552a8aa9f0154f868fc026e6015a3c1ffb722/doc/s2n_quic/provider/tls/rustls/index.html.

It might be worth doing the same so applications can be using the same version we are.